python/doc/examples/probabilistic_modeling/random_vectors/plot_ratio_of_uniforms_exploitation.py

```diff
@@ -0,0 +1,455 @@
+view = otv.View(grid_default)
+
+# %%
+# We change the :math:`r` value and we draw the same plots.
+grid_full = ot.GridLayout(4, 2)
+for i, r in enumerate([0.5, 0.8, 1.2, 2.0]):
+    ratio_Algo.setR(r)
+    g_UV = draw_RatioOfUniforms_UV(ratio_Algo)
+    g_UV.setLegendPosition("")
+    g_UV.setTitle(
+        rf"$r = ${ratio_Algo.getR():.3g}, $\tau = ${ratio_Algo.getAcceptanceRatio():.3g}"
+    )
+
+    g_XY = draw_RatioOfUniforms_XY(ratio_Algo, 1001)
+    g_XY.setLegendPosition("")
+    g_XY.setTitle(
+        rf"$r = ${ratio_Algo.getR():.3g}, $\tau = ${ratio_Algo.getAcceptanceRatio():.3g}"
+    )
+
+    grid_full.setGraph(i, 0, g_UV)
+    grid_full.setGraph(i, 1, g_XY)
+
+grid_full.setTitle(r"$\log f(x) = \log(\cos(x)) + x,  \quad x \in ]-\pi/2, \pi/2[$")
+view = otv.View(grid_full)
+
+# %%
+# Study of the acceptance ratio.
+# ------------------------------
+# We study the acceptance ratio :math:`\tau(r)` as a function of the parameter :math:`r` for some distributions.
+# In these cases, the value is exact and computed from the volume of :math:`A_{f,r}` and the one of :math:`\tilde{A}_{f,r}`.
+# We draw the function :math:`r \rightarrow \tau(r)`. We conclude that the default value :math:`r=1` seems to be correct
+# in all these cases.
+#
+mixt_gaus = ot.Mixture([ot.Normal(-2.0, 1.0), ot.Normal(4.0, 1.0)], [0.2, 0.8])
+logPDF_list = [
+    ot.Normal().getLogPDF(),
+    ot.Normal(2).getLogPDF(),
+    ot.Normal(3).getLogPDF(),
+    ot.Exponential().getLogPDF(),
+    mixt_gaus.getLogPDF(),
+]
+range_list = [
+    ot.Normal().getRange(),
+    ot.Normal(2).getRange(),
+    ot.Normal(3).getRange(),
+    ot.Exponential().getRange(),
+    mixt_gaus.getRange(),
+]
+name_list = ["Normal(1)", "Normal(2)", "Normal(3)", "Exponential()", "Mixture Normal"]
+
+g = ot.Graph(
+    r"Acceptance ratio $\tau$ as a function of $r$", "r", r"$\tau$", True, "topright"
+)
+g.setLogScale(1)
+numberR = 50
+a = (1e2) ** (2.0 / numberR)
+data_r_ratio = ot.Sample(numberR + 1, 2)
+isScaled = True
+for k in range(len(logPDF_list)):
+    ratio_Algo = otexp.RatioOfUniforms(logPDF_list[k], range_list[k], isScaled)
+    for i in range(numberR + 1):
+        r = a ** (i - 0.5 * numberR)
+        ratio_Algo.setR(r)
+        accept_Ratio = ratio_Algo.getAcceptanceRatio()
+        data_r_ratio[i] = [r, accept_Ratio]
+    draw = ot.Curve(data_r_ratio)
+    draw.setLegend(name_list[k])
+    g.add(draw)
+
+isScaled = False
+ratio_Algo = otexp.RatioOfUniforms(log_UnscaledPDF, range_PDF, isScaled)
+for i in range(numberR + 1):
+    r = a ** (i - 0.5 * numberR)
+    ratio_Algo.setR(r)
+    accept_Ratio = ratio_Algo.getAcceptanceRatio()
+    data_r_ratio[i] = [r, accept_Ratio]
+draw = ot.Curve(data_r_ratio)
+draw.setLegend(
+    r"$f(x) = \cos(x)e^x\mathbf{1}_{\left[-\frac{\pi}{2}, \frac{\pi}{2}\right]}(x)$"
+)
+g.add(draw)
+
+g.setLegendPosition("upper left")
+g.setLegendCorner([1.0, 1.0])
+view = otv.View(g)
+
```

Comment:
idem adding a short sentence of analysis of the above plot on the accpetance ratio as a function of r would help the reader that are not familiar with such algorithm and the key role of the parameter r